contracts/Trustcoin.sol

```diff
@@ -21,7 +21,7 @@ contract Trustcoin is OutgoingMigrationTokenInterface, ERC20TokenInterface, Safe
   string public constant symbol = 'TRST';
   string public constant version = 'TRST1.0';
   uint256 public constant minimumMigrationDuration = 26 weeks; // Minumum allowed migration period
-  uint256 public totalSupply = 100000000; // One hundred million (ERC20)
+  uint256 public totalSupply = 100000000 * (10 ** decimals); // One hundred million (ERC20)
```

Comment:
good catch!

Unrelated: I'm wondering if a decimal of 18 is not too much and will just make it unfriendly to use.
Suppose the market cap of the coins is $100B, then we'll only need 3 decimals for 100M coins.
Can we do with, say, 6 decimals?
The fact ETH is using some number is not in itself a strong reason to use the same number.